### DISCUSSION TITLE
Where can i edit the settings?

### DISCUSSION THREAD

**Discussion by 99thGamer9** [GriefPrevention/GriefPrevention#1383] [Q&A] [answered]
I cant find them in global.conf or in-game. Also is there a difference between the bukkit and sponge version?

**Comment by RoboMWM** [comment]
This is a bukkit plugin. There is no "conf" file.

**Reply by 99thGamer9** [reply]
I'm using the sponge version

**Reply by RoboMWM** [reply]
The sponge version has widely diverged from the original Bukkit version and is not supported here.

**Comment by bigpresh** [comment]
Look in your server's dir under `plugins/GriefPreventionData/config.yml`

**Reply by 99thGamer9** [reply]
There is no plugins folder when using sponge, but i guess few people even use the sponge version

**Comment by 99thGamer9** [comment, marked as answer]
I found the answer, you need to change them using your permission plugin

**Reply by bigpresh** [reply]
Well, the proper answer, if you're using the Sponge version, is to ask your question on the Sponge version's GitHub (or wherever they provide support) as this repo is the Bukkit one :)  Glad you found it though and thanks for taking the time to report the solution for anyone who finds this by searching in future, kudos!
